src/main/java/com/skhu/luxuryshop/product/dto/ProductResponseDto.java

```diff
@@ -0,0 +1,28 @@
+package com.skhu.luxuryshop.product.dto;
+
+import com.skhu.luxuryshop.product.entity.ProductEntity;
+import lombok.Getter;
+
+@Getter
+public class ProductResponseDto {
+    private long id;
```

Comment:
```suggestion
    private Long id;
```

id를 Long으로 했을 때 장점 저번에 리뷰달아드린거 기억하시죠! 
그거 다시 봐보시면 좋을 듯해요!  DTO에도 적용되는 장점이 있슴다!